lib/schema/translation_tooling.rb

```diff
@@ -0,0 +1,56 @@
+require 'yaml'
+
+fact_schema = YAML.load_file(File.join(File.dirname(__FILE__), "facter.yaml"))
+facts = []
+
+fact_schema.each do |fact|
+  fact_name = fact[0]
+  fact_details = fact[1]
+
+  fact_data = {:name => fact_name,
+               :description => fact_details['description'],
+               :resolution => fact_details['resolution'],
+               :caveats => fact_details['caveats']}
+
+  facts << fact_data
+end
+
+File.open(File.join(File.dirname(__FILE__), "core_facts.pot"), 'w') do |file|
```

Comment:
Do we want the output file to live down in lib/schema/core_facts.pot or possibly up in the top-level locales/ directory? Perhaps @LarissaLane or @MikaelSmith have an opinion?